src/DOMTokenList.js

```diff
@@ -72,10 +72,13 @@ module.exports = class DOMTokenList extends Array {
   }
 
   replace(token, newToken) {
-    const i = this.indexOf(token);
-    if (i < 0) this.add(newToken);
-    else this[i] = newToken;
-    afterChanges(this);
+    for (let i = 0, n = this.length; i < n; i++) {
```

Comment:
I cannot see anything in that link but regardless indexOf is one word to maintain VS few lines so this change won't land in this project